src/prototypes/example/styles.scss

```diff
@@ -1,3 +1,21 @@
-.header__main {
-  background: #333;
+/***********************************************************************
+Update the version in the path below to the version being used
+***********************************************************************/
+@import '../../../node_modules/@ons/design-system/45.1.3/scss/vars/_index.scss';
+@import '../../../node_modules/@ons/design-system/45.1.3/scss/helpers/_index.scss';
+@import '../../../node_modules/@ons/design-system/45.1.3/scss/utilities/_index.scss';
+
+/***********************************************************************
+Do not remove. Required for dynamic generation of survey summary page
+***********************************************************************/
+.ons-summary__item:not(:last-child) {
+  border-bottom: 1px solid $color-borders !important;
 }
+
+.js-summary {
```

Comment:
```suggestion
.ons-js-summary {
```

Reply:
@jrbarnes9 We don't need to do this as it's just for the prototype kit.